tests/Feature/Controllers/Api/Journals/Modules/Reading/ReadingControllerTest.php

```
@@ -0,0 +1,155 @@
+        $entry->refresh()->load('moduleReading');
+        $this->assertEquals('deep immersion', $entry->moduleReading->reading_amount);
+    }
+
+    #[Test]
+    public function it_validates_reading_amount_must_be_valid(): void
+    {
+        $user = User::factory()->create();
+        $journal = Journal::factory()->create([
+            'user_id' => $user->id,
+        ]);
+        $entry = JournalEntry::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$entry'. (undefined)

(UnusedLocalVariable)